### ISSUE TITLE
How exaclty is the global triplet loss function implemented? Can you provide an implementation of it?

### ISSUE THREAD

**Issue by kaushal-py** [vijaykbg/global-patchmatch#1]

**Comment by rteja1113** [comment]
Yeah, I'd like to know the answer for that too, since the paper says global losses are computed using the 'entire training data', which means it should have access to activations of entire training set during backprop. This seems impossible given the limited amount of memory in GPU.

or does 'entire training set' mean the current mini-batch ?. I'm not sure what 'N' is in the paper.

**Comment by learnxy** [comment]
d

**Comment by learnxy** [comment]
yeah i also want to know what's mean of local image? it's one image for the total training set or just a patch for one image??

**Comment by swg209** [comment]
@kb-studios  Hi, had you implement the code of global loss function?
